### ISSUE TITLE
MSFS 2024 Compatibility

### ISSUE THREAD

**Issue by Togapowerr** [Fragtality/DynamicLOD#15]
Hello, I absolutely love DynamicLOD (I used the slightly tweaked version by ResetXPDR) and have consider it an essential tool for MSFS2020. It has transformed the experience for me as it's allowed me to have great frames on the ground and in landing phases of flight, while being able to enjoy high terrain LODs at cruise altitude.

Are there any plans to make this work with FS24?

**Comment by Fragtality** [comment]
Well, let's say I hope to never have any Plans to make this work for MSFS2024! I hope that the Sim is now itself intelligent enough to not waste Time on Terrain/Objects the User can't see anyways. 

Besides that, the Core-Mechanic how to change the Sim's xLOD Settings on the Fly was a Discovery by Corsten & muumimorko. ResetXPDR and myself "just" build on that Foundation and do not have any Clue how they discovered that.

**Comment by Togapowerr** [comment]
MSFS2024 does have a Dynamic settings function, but I have yet to see how well it works considering the nature of the release and major bugs the game is currently experiencing. I guess we'll see!

Nonetheless, I really appreciate the work you put in for this mod and, once again, it has transformed my MSFS2020 experience. Cheers to you!

**Comment by ResetXPDR** [comment]
> Besides that, the Core-Mechanic how to change the Sim's xLOD Settings on the Fly was a Discovery by Corsten & muumimorko. ResetXPDR and myself "just" build on that Foundation and do not have any Clue how they discovered that.

That is correct. I did have a quick look when the tech Alpha was active to see if it was straightforward to get the new memory map location of MSFS 2024 settings. It was not, so unless someone has the skills to do this for us we are pretty much dead in the water with 2024 compatibility.

**Comment by Fragtality** [comment]
Done.
